### DISCUSSION TITLE
Top bar not colored anymore in darkmode?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#627] [General]
Is this by design or a bug? In Dark mode the top bar is now always dark.

**Comment by razonyang** [comment]
Yep, the colored top app bar is too bright in dark mode. I can add a parameter for keeping the colored on dark if you want.

**Comment by Natenom** [comment]
Yep, a paramater would be nice as I would like to always have "my" blue color on the website :)

**Comment by razonyang** [comment]
Done via #629
